rituals/kevin.md

```diff
@@ -0,0 +1,26 @@
+# Kevin
+
+## Time Based
+
+### Daily
+* Start Early.
+* Check emails, Slack, and Asana (maybe DataDog/Sentry/Change Requests as I get familiar) for anything that needs to be addressed.
+* Work on things that need to be addressed.
+* Work on existing ticket: code, PR feedback, etc.
+* Review team/team adjacent PRs.
+* See if anyone needs/can pair.
+* Lunch
+* Standup
+* Continue pairing/coding/etc.
+* Pickup new work as needed
+
+### Weekly
+* Think about 1-on-1 talking points.
+* Note any retro items.
+
+### Monthly
+* Re-read / Review guides and documentation
+* Check-in on personal rituals and adjust as needed
+
+## Pattern Based
```

Comment:
Is the idea that some of us post these here? Should we do in our Notion spaces or elsewhere?

Reply:
I guess this question goes to Anthony, TIL we have that directory! 😁

Reply:
I'm good to remove this, I have no skin in this game, just saw it. I like the notion idea. @acostanzo ?

Reply:
This is so wildly out dated for me anyway. I'm ok with dropping it